agent/php_execute.c

```diff
@@ -874,6 +880,30 @@ static void nr_execute_handle_library(const char* filename TSRMLS_DC) {
   nr_free(filename_lower);
 }
 
+static void nr_execute_handle_logging_framework(const char* filename TSRMLS_DC) {
```

Comment:
Small nit but maybe should be called "nr_execute_handle_logging_frameworks()?".

Reply:
I think the intention of why this set of functions: `nr_execute_handle_framework`, `nr_execute_handle_library`,  `nr_execute_handle_logging_framework` end in singular form of the noun is because they operate on one file that is being executed and they try to handle if the file is part of a framework/library. I'd stick with singular form. I don't mind changing the `_logging_framework` part to `_logging_library` - in PHP world Monolog is called a 'package' anyway :)

Reply:
Ah that makes sense for the singular. Thanks!